tensorflow/stream_executor/gpu/asm_compiler.cc

```diff
@@ -237,6 +249,16 @@ port::StatusOr<std::vector<uint8>> CompileGpuAsm(int cc_major, int cc_minor,
   return cubin_vector;
 }
 
+char* GetTFExtraPTXOptions() {
+  static bool fetched = false;
+  static char* env = nullptr;
+  if (!fetched) {
+    env = getenv("TF_EXTRA_PTXAS_OPTIONS");
```

Comment:
If it's only for XLA, the flag should go through XLA_FLAGS. See https://github.com/tensorflow/tensorflow/blob/13645c7fd2a7c151b82a9bfa144b0e682122ed7b/tensorflow/compiler/xla/xla.proto#L25

`GpuAsmOpts` already has a member `extra_flags`, which can be plumbed through XLA_FLAGS.